openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: val
+oeps: {}
+owner: edx/teaching-and-learning
```

Comment:
Thinking about this practically, it would be better as edx/teaching-and-learning. I don't think mobile will be touching this even if we did 'own' this.